aws-xray-recorder-sdk-log4j/build.gradle.kts

```diff
@@ -6,9 +6,9 @@ plugins {
 dependencies {
     api(project(":aws-xray-recorder-sdk-core"))
 
-    compileOnly("org.apache.logging.log4j:log4j-api:2.13.3")
+    compileOnly("org.apache.logging.log4j:log4j-api:2.15.0")
```

Comment:
Just in case don't know / helps - you shouldn't need to do a patch release for this since it's a `compileOnly` dependency